STL_Extension/include/CGAL/In_place_list.h

```diff
@@ -171,6 +177,12 @@ namespace internal {
     {
       return In_place_list_iterator<T,Alloc>(const_cast<T*>(node));
     }
+
+    friend std::ostream& operator<<(std::ostream& os, const Self& i)
+    {
+      os << &*(i);
+      return os;
```

Comment:
`&*i` triggers an undefined behavior if the iterator is singular (points to `nullptr`).

```suggestion
      return os << i.operator->();
```